module-2/lab-pivot-table-and-correlation/your_code/main.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
```

Comment:
- You don't need to use the following to show the plot. I think here is your syntax error. However, you can also use matplotlib instead of seaborn for this exercise. Please, review it and complete this part of the assignment when you have some free time. 
```Python
fig, ax = plt.subplots()
```